docs/tutorials/t2ramsey_characterization.ipynb

```diff
@@ -13,9 +13,9 @@
    "source": [
     "This experiment serves as one of the series of experiments used to characterize a single qubit. Its purpose is to determine two of the qubit's properties: *Ramsey* or *detuning frequency* and $T_2\\ast$. The rough frequency of the qubit was already determined previously. The control pulses are based on this frequency. \n",
```

Comment:
Should this sentence be kept? It's not in the T1 tutorial and I would prefer they have the same style.